### ISSUE TITLE
multiple prometheus servers that collide on port binding

### ISSUE THREAD

**Issue by gabrieljones** [open-telemetry/opentelemetry-java#5810]
**Describe the bug**
`AutoConfiguredOpenTelemetrySdk.builder().build()` starts multiple prometheus servers that collide on port binding.

**Steps to reproduce**
With system properties of `otel.metrics.exporter=prometheus`
Call `AutoConfiguredOpenTelemetrySdk.builder().build()`

**What did you expect to see?**
A single prometheus server instance

**What did you see instead?**
A second prometheus server startup that caused a `java.net.BindException: Address already in use`

**What version and what artifacts are you using?**

```
io.opentelemetry:opentelemetry-sdk-extension-autoconfigure:1.30.0
io.opentelemetry:opentelemetry-exporter-prometheus:1.30.0-alpha
```

```build.gradle.kts
//build.gradle.kts
//...
dependencies {
  //...
  implementation(platform("io.opentelemetry:opentelemetry-bom:latest.release"))
  implementation("io.opentelemetry:opentelemetry-sdk")
  runtimeOnly("io.opentelemetry:opentelemetry-exporter-logging")
  runtimeOnly("io.opentelemetry:opentelemetry-exporter-otlp")
  runtimeOnly("io.opentelemetry:opentelemetry-exporter-prometheus")
  runtimeOnly("io.opentelemetry:opentelemetry-exporter-otlp-logs:latest.integration")
  implementation(platform("io.opentelemetry:opentelemetry-bom-alpha:latest.integration"))
  implementation("io.opentelemetry:opentelemetry-semconv")
  implementation("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure")
  implementation("io.opentelemetry.instrumentation:opentelemetry-kafka-clients-2.6:latest.integration")
  runtimeOnly("io.opentelemetry.instrumentation:opentelemetry-log4j-appender-2.17:latest.integration")
  //...
}
```

**Environment**
```
> ./gradlew -version

------------------------------------------------------------
Gradle 8.3
------------------------------------------------------------

Build time:   2023-08-17 07:06:47 UTC
Revision:     8afbf24b469158b714b36e84c6f4d4976c86fcd5

Kotlin:       1.9.0
Groovy:       3.0.17
Ant:          Apache Ant(TM) version 1.10.13 compiled on January 4 2023
JVM:          17.0.8 (Eclipse Adoptium 17.0.8+7)
OS:           Mac OS X 13.4 x86_64
```

**Comment by gabrieljones** [comment]
A first instance is started deep inside:
https://github.com/open-telemetry/opentelemetry-java/blob/v1.30.0/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java#L348-L349

A second one is started somewhere in:
https://github.com/open-telemetry/opentelemetry-java/blob/v1.30.0/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java#L350

**Comment by jack-berg** [comment]
Yup, we just noticed this as well and opened #5811 to address it.

Will be releasing a patch with the fix.

**Comment by jack-berg** [comment]
Fixed in patch release [1.30.1](https://github.com/open-telemetry/opentelemetry-java/releases/tag/v1.30.1).
